<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubmitFormClass</class>
 <widget class="QWidget" name="SubmitFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SubmitForm</string>
  </property>
  <property name="windowIcon">
   <iconset resource="AutomaticStart.qrc">
    <normaloff>:/仓鼠.ico</normaloff>:/仓鼠.ico</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="planName">
     <property name="styleSheet">
      <string notr="true">font: 12pt '仿宋';</string>
     </property>
     <property name="text">
      <string>计划名称</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTextEdit" name="textEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="services">
     <property name="styleSheet">
      <string notr="true">font: 12pt '仿宋';</string>
     </property>
     <property name="text">
      <string>服务列表</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="addService">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: bold 12pt '仿宋';</string>
       </property>
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="deleteService">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: bold 12pt '仿宋';</string>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QTableWidget" name="serviceList"/>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="submit">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: bold 12pt '仿宋';</string>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AutomaticStart.qrc"/>
 </resources>
 <connections/>
</ui>
